coreneuron/utils/hash.hpp

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
this file is unused? remove this

Reply:
yeah, cannot work properly with constexpr in C++11, so for compatibility, we cannot use that, I remove it.